kong/clustering/rpc/manager.lua

```diff
@@ -339,13 +386,12 @@ function _M:connect(premature, node_id, host, path, cert, key)
     ssl_verify = true,
     client_cert = cert,
     client_priv_key = key,
-    protocols = "kong.rpc.v1",
+    protocols = RPC_MATA_V1,
     headers = {
       "X-Kong-Version: " .. KONG_VERSION,
       "X-Kong-Node-Id: " .. self.node_id,
       "X-Kong-Hostname: " .. kong.node.get_hostname(),
```

Comment:
Let's move these into the meta call as well.

Reply:
done, but `node_id` is necessary for rpc framework, see:
```
local s = socket.new(self, wb, node_id)
```